.github/workflows/build-and-run-model.yaml

```diff
@@ -45,16 +45,26 @@ on:
         description: Run cross-validation
         default: false
         required: true
-      comp_enable:
+      shap_enable:
         type: boolean
-        description: Run comparables finding
+        description: Calculate SHAP values
         default: false
         required: true
-      shap_enable:
+      comp_enable:
         type: boolean
-        description: Calculate SHAP values
+        description: Run comparables finding
         default: false
         required: true
+      num_comps:
+        type: string
+        description: Integer number of comps to look for
+        default: "20"
+        required: false
+      num_comp_price_bins:
+        type: string
+        description: Integer number of price bins to use for comps finding
+        default: "30"
+        required: false
```

Comment:
I shuffled things around a bit to try to group the comps configs together and avoid having them break up the rest of the boolean configs. Here's what the menu looks like after this change:

![Screenshot 2024-01-26 150025](https://github.com/ccao-data/model-res-avm/assets/14170650/0ecff16c-2d46-40ea-a282-d3c75fa15a92)

It feels like it's getting a bit unwieldy, so I'm curious if you have feedback! It would be nice if we could hide `num_comps` and `num_comp_price_bins` and only display them when `comp_enable` is selected, but unfortunately it doesn't seem like workflow dispatch supports that type of form logic yet.

Reply:
**issue (blocking)** Oh oops, I think I was unclear: we don't need these adjustable in the dispatched workflow, only settable via a key in `params.yaml`. They shouldn't change often enough to warrant a dispatch field.

Reply:
Ah I see now, that makes a lot more sense! Done in e4ec5a383621fc51b49169a8017c5082bcde43fd 👍🏻